R/latex_dependencies.R

```diff
@@ -1,9 +1,10 @@
 #' Define a LaTeX package dependency
 #' @param name The LaTeX package name
 #' @param options The LaTeX options for the package
+#' @param ... Extra LaTeX code related to the package added to the preamble
 #' @export
-latex_dependency <- function(name, options = NULL) {
-  output <- list(name = name, options = options)
+latex_dependency <- function(name, options = NULL, ...) {
```

Comment:
Could you just use `extra_lines` as the argument instead of `...`? Using a general `...` argument will make it slightly difficult for us to add other possible arguments to this function in the future.

Reply:
Makes sense!